app/src/constants/i18n.ts

```diff
@@ -13,3 +13,11 @@ export const UploadProjectArtifactsI18N = {
   uploadErrorText:
     'An error has occurred while attempting to upload project artifacts, please try again. If the error persists, please contact your system administrator.'
 };
+
+export const EditObjectivesI18N = {
+  editTitle: 'Edit Project Objectives',
+  editText: 'Are you sure you want to save?',
+  createErrorTitle: 'Error Editing Project Objectives',
+  createErrorText:
```

Comment:
I don't see these being used anywhere - are they supposed to be?

Reply:
yeah we need to add them, but we don't have error handling on this PR